src/platforms/dotnet/migration.mdx

```diff
@@ -108,3 +108,183 @@ A new package was added ASP.NET Core gPRC support. With the ability to log the r
 # Release notes
 
 Please refer to [GitHub for the complete release notes](https://github.com/getsentry/sentry-dotnet/releases/tag/3.0.0).
+
+<Note>
+
+Migrating from SharpRaven to Sentry SDK
+
+</Note>
+
+# Breaking changes
```

Comment:
Two things here (please make these changes throughout):
1. Headings use Title Case
2. Headings need to be followed by at least one line of text (or the content for "On this page" doesn't work properly)

```suggestion
# Breaking Changes

Upgrading from SharpRaven to Sentry SDK includes the following breaking changes.
```

Reply:
> Headings use Title Case

Only h1 or all of them?

Reply:
All (and I know some content isn't yet updated to this standard, so I'm glad you asked).